Tests/VSMTests/StateContainerTests.swift

```diff
@@ -4,82 +4,8 @@ import XCTest
 import VSM
 
 class StateContainerTests: XCTestCase {
-    
-    // MARK: - Publisher Actions
-    
-    /// Asserts that observing state-emitting publisher actions will progress the state appropriately
-    func testStatePublisherAction_MultipleStates_Immediate() throws {
```

Comment:
These tests have been moved to their respective protocol tests and optimized for performance and accuracy.